starcounter-include.html

```diff
@@ -32,13 +32,74 @@
             static get observedAttributes() {
                 return ['partial', 'view-model', 'partialId'];
             }
-            // the self argument might be provided or not
-            // in both cases, the mandatory `super()` call
-            // will return the right context/instance to use
-            // and eventually return
-            constructor(self) {
-                self = super(self);
-                this.createComponent();
+            /**
+            * Create shadowRoot, define property setters, set initial partial.
+            */
+            constructor() {
+                super();
+                this.defaultComposition = null;
+                var partialId = this.partialId;
```

Comment:
This line seems unused.

Reply:
It is used. It declares the scope for private `partialId` used in here https://github.com/Starcounter/starcounter-include/blob/020bea936c4880bd9b4c761fbbff795793c66d03/starcounter-include.html#L90

Reply:
Also it allows to set this property before element is upgraded.

Reply:
Can we do this somehow differently, e.g. `this.partial = this.partial` at the end of the constructor?